<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>158</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>新建账单小助手</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_5">
      <property name="font">
       <font>
        <family>Agency FB</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
        <underline>true</underline>
       </font>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ff007f;&quot;&gt;新建账单小助手v2.2&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>新的账单名：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit">
        <property name="placeholderText">
         <string>输入想要创建的账单名称</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>.xlsx</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>生成账单</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>工期段：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox_3">
        <property name="editable">
         <bool>true</bool>
        </property>
        <item>
         <property name="text">
          <string>2021</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2022</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2023</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2024</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2025</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2026</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2027</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2028</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2029</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2030</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>年</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox_2">
        <item>
         <property name="text">
          <string>01</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>02</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>03</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>04</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>05</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>06</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>07</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>08</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>09</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>10</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>11</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>12</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>月</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox_1">
        <item>
         <property name="text">
          <string>01</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>02</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>03</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>04</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>05</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>06</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>07</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>08</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>09</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>10</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>11</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>12</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>13</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>14</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>15</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>16</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>17</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>18</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>19</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>20</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>21</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>22</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>23</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>24</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>25</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>26</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>27</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>28</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>29</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>30</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>31</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>日-</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox_4">
        <property name="editable">
         <bool>true</bool>
        </property>
        <item>
         <property name="text">
          <string>2021</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2022</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2023</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2024</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2025</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2026</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2027</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2028</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2029</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2030</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>年</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox_5">
        <item>
         <property name="text">
          <string>01</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>02</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>03</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>04</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>05</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>06</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>07</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>08</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>09</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>10</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>11</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>12</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>月</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox_6">
        <item>
         <property name="text">
          <string>01</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>02</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>03</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>04</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>05</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>06</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>07</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>08</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>09</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>10</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>11</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>12</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>13</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>14</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>15</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>16</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>17</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>18</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>19</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>20</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>21</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>22</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>23</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>24</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>25</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>26</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>27</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>28</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>29</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>30</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>31</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>日</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QPushButton" name="pushButton_2">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>写入工期</string>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_3">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>退出</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="yiyan_label">
        <property name="font">
         <font>
          <family>楷体</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>没有检测到新版本!</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QProgressBar" name="progressBar">
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
